consensus/module.go

```diff
@@ -116,7 +138,39 @@ func Create(cfg *genesis.Config, genesis *genesis.GenesisState) (modules.Consens
 	return m, nil
 }
 
-func (m *consensusModule) Start() error {
+func (m *ConsensusModule) InitConfig(pathToConfigJSON string) (config modules.ConfigI, err error) {
+	data, err := ioutil.ReadFile(pathToConfigJSON)
+	if err != nil {
+		return
+	}
+	// over arching configuration file
+	rawJSON := make(map[string]json.RawMessage)
+	if err = json.Unmarshal(data, &rawJSON); err != nil {
+		log.Fatalf("[ERROR] an error occurred unmarshalling the config.json file: %v", err.Error())
```

Comment:
Replace with: `log.Fatalf("[ERROR] an error occurred unmarshalling the %s file: %v", pathToConfigJSON , err.Error())`

Ditto for every other error in config and JSON unmarshalling, but only leaving this one comment